src/main.cpp

```diff
@@ -3389,6 +3403,13 @@ struct PoolMetrics {
         return stats;
     }
 
+    static PoolMetrics Orchard(CBlockIndex *pindex, CCoinsViewCache *view) {
+        PoolMetrics stats;
+        stats.value = pindex->nChainOrchardValue;
```

Comment:
What about stats.created, or the other fields?

Reply:
Can't add those yet, we don't have the commitment tree.